src/main/java/org/tron/common/runtime/vm/program/Program.java

```diff
@@ -1506,7 +1506,12 @@ public StackTooLargeException(String message) {
   }
 
   public DataWord getCallEnergy(OpCode op, DataWord requestedEnergy, DataWord availableEnergy) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused method parameter "op". ([squid:S1172](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1172))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1633&issues=AWak_C1ndPPRVdwyhXJ8&open=AWak_C1ndPPRVdwyhXJ8)

[ISSUE_KEY]: AWak_C1ndPPRVdwyhXJ8